packages/app/background-jobs-common/src/background-job-processor/index.ts

```diff
@@ -7,3 +7,4 @@ export * from './processors/types'
 export * from './processors/AbstractBackgroundJobProcessor'
 export * from './processors/FakeBackgroundJobProcessor'
 export * from './spy/types'
+export { sanitizeRedisConfig, createSanitizedRedisClient } from './utils'
```

Comment:
Stupid question, but where would we use this? just wondering if we really need this exposed

Reply:
One example is in fastify-extras, we have 2 queue discoverers there: one uses `backgroundJobProcessorGetActiveQueueIds` from `background-jobs-common`, another one operates directly on redis instance, and there we could still benefit from sanitized instance instead of creating one with `new Redis`.

Also, in Flow, we always create Redis manually from config, like: `new Redis(redisConfig)`. I think in the follow up PR's we could replace it with `createSanitizedRedisClient` to ensure that required properties are always there.

Does it make sense?

Reply:
For Fastify extras, I agree completely and it makes a lot of sense.

On Flow side, I am not sure if we want to sanitaise redis config, as it is a requirement for BullMQ and the library is already taking care that those connections are sanitized